core/server/apps/subscribers/lib/helpers/index.js

```diff
@@ -1,19 +1,18 @@
 // Dirty requires!
-var labs = require('../../../../services/labs');
+const labs = require('../../../../services/labs');
 
 module.exports = function registerHelpers(ghost) {
     ghost.helpers.register('input_email', require('./input_email'));
 
     ghost.helpers.register('subscribe_form', function labsEnabledHelper() {
-        var self = this,
-            args = arguments;
+        let self = this, args = arguments;
 
         return labs.enabledHelper({
             flagKey: 'subscribers',
             flagName: 'Subscribers',
             helperName: 'subscribe_form',
             helpUrl: 'https://help.ghost.org/hc/en-us/articles/224089787-Subscribers-Beta'
-        }, function executeHelper() {
+        }, () => {
```

Comment:
In this case it's not an anonymous function. And if we would like to loose the named function, we no longer need to apply this and the arguments when using arrow functions.

Reply:
@kirrg001 I was asking about named functions in these situations in original issue (#9589). What's purpose for names here and if we could change them to arrow functions if the function name was not used.

Reply:
We can keep the arrow function here. We have encounter any problems with debugging, we have to think about arrow vs none arrow again.

When using arrow functions here, i think we can get rid of `self = this` and use `this` directly.